client/client.ts

```diff
@@ -195,13 +217,14 @@ async function spawnSignatureResponse(
 
     console.log("Submitting spawn proof to nStates");
     try {
-        await nStates.write.spawn([
+        const tx = await nStates.write.spawn([
             spawnInputs,
             spawnProof,
             virtInputs,
             virtProof,
             spawnSig,
         ]);
+        console.log("spawn tx hash", tx);
```

Comment:
Moving toward formatting prod logs like this:
"Submitting spawn proof to nStates" -> "- Submitting spawn proof to NS contract"
"spawn tx hash" -> "- Spawn proof tx hash:"

the dash and proper capitalization / puncutation helps us differentiate between actual logs and debug outputs